_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,147 @@
+Doing so helped identify what worked well and what challenges surfaced.
+It also helped figure out what mechanisms of control should be added to ensure that the organization kept a good hold on its information and how to better support their employees in adopting the best practices.
+
+If your team isn't fully ready to work in the open, try slowly opening up your workspace to the rest of the organization by default.
+Don't wait until "that presentation is ready" before letting it be discoverable by others.
+You know you will not get around to it!
+
+## Set up your environment
+
+Your virtual workspace may consist in a variety of tools and platforms.
+
+Software developers and IT folks around the world may leverage social coding platforms such as Gitlab, Github, GCcode, etc., to collaborate widely with people around the world on source code.
+
+In the Government of Canada, most departments now use Microsoft 365 to varying degrees, which may include Teams, a collaboration platform that ties in multiple products and capabilities.
+
+Whether you are using the corporate collaboration platform or a 3rd party software as service to work in the open, there are a number of things to keep in mind.
+
+### Stay in control
+
+Your workspace is a very important place: it contains your work and you have a responsibility to ensure its properly secured and controlled.
+However, that doesn't mean you can't open up securely.
+
+A proper access management is required so that we can separate the two main types of users: your team's members and external contributors.
+The latter could be a person from within your organization or from the public.
+
+There may be more types of roles available depending on the platforms, but these are the main buckets we think of when working in the open.
+
+Depending on the platform you use, various controls may be offered to let you share openly your documents and your work while remaining in control of its content.
+
+In SharePoint, most often available as "Inner Source" only, a team can open their workspace by default and create specific folders with more restricted access.
+
+By default, all folders and document should be set to be open to the whole organization, only creating folders and documents privately when really required.
+
+One of the option you may want to choose is to only approve comments from external contributors.
+This way, people can stop by and provide feedback while you're working but still remain in control of the main version of the document.
+
+### Ensure privacy
+
+Citizen data is a critical element public servants may have to work with during their career in the government and it must never be shared publicly.
```

Comment:
I wonder if this is better as 'Public servants who work with citizen data must take great care regarding the privacy of that data and never share it publicly.'.  The 'critical element' wording gets me stuck...

Reply:
I like the rewording!